### PR TITLE
Add cmd sleep

### PR DESCRIPTION
**Pull request by kasturinarra**

### CODE REVIEW

File: lib/rules/cli/4.1.yaml [hunk 1]
```diff
@@ -186,6 +186,7 @@
     :restart: --restart=<value>
     :save_config: --save-config=<value>
     :schedule: --schedule=<value>
+    :sleep:  -- sleep <value>
```
Comment (pruan-rht) [inline lib/rules/cli/4.1.yaml:189]: @kasturinarra should it be `--sleep=<value>`

File: lib/rules/cli/4.4.yaml [hunk 2]
```diff
@@ -186,6 +186,7 @@
     :restart: --restart=<value>
     :save_config: --save-config=<value>
     :schedule: --schedule=<value>
+    :sleep:  -- sleep <value>
```
Comment (pruan-rht) [inline lib/rules/cli/4.4.yaml:189]: ditto...

File: lib/rules/cli/4.1.yaml (hunk 1 above)
Comment (kasturinarra) [reply lib/rules/cli/4.1.yaml:189]: It should not have '=',  the command which i wanted to run is "oc create cronjob <name> <imagename> --schedule='* * * * *' -- sleep 180

Comment (pruan-rht) [reply lib/rules/cli/4.1.yaml:189]: @kasturinarra if `:sleep` is not part of the `oc command` then this shouldn't be part of the yaml file.  We only add options that are specific to the `oc` client command

**Comment by kasturinarra** [comment]
closing this due to https://github.com/openshift/verification-tests/pull/1095

**Pull request closed without merging**